addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1484,10 +1485,17 @@ export class OdooEditor extends EventTarget {
         });
         if (!range) return;
         let { startContainer: start, startOffset, endContainer: end, endOffset } = range;
+        const startBlock = closestBlock(start);
+        const endBlock = closestBlock(end);
+        // Do not join blocks in the following cases:
+        // 1. start and end share a common ancestor block with the range
+        // 2. selection spans multiple TDs
+        // 3. selection starts at beginning of startBlock and ends at end of endBlock
```

Comment:
I counted it, and its **77**, 3 less than 80 chars comment limit. :sweat_smile:

Reply:
78 + the 8-char indentation so 86:
![image](https://github.com/odoo/odoo/assets/24205914/30ca1c69-049d-4be2-9524-4bec82c64d89)